src/scripts/Player.ts

```diff
@@ -29,6 +29,7 @@ class Player {
         const saved: boolean = (savedPlayer != null);
         savedPlayer = savedPlayer || {};
         this._lastSeen = savedPlayer._lastSeen || 0;
+        this._previousStartLastSeen = savedPlayer._lastSeen || 0;
```

Comment:
I think instead of adding this, we should just not bother updating the `_lastSeen` variable to the current time in the constructor below.

As it's already updated on save anyway, and not used anywhere else.